internal/handlers/pg/msg_delete.go

```diff
@@ -153,20 +152,21 @@ func (h *Handler) MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return err
 	}
 
-	var unorderedErrMsgs []string
+	var lastErr error
 
 	for i := 0; i < deletes.Len(); i++ {
 		err := processQuery(i)
 		if err != nil {
 			if ordered {
 				return nil, err
 			}
-			unorderedErrMsgs = append(unorderedErrMsgs, err.Error())
+			lastErr = err
 		}
 	}
 
-	if len(unorderedErrMsgs) > 0 {
-		return nil, fmt.Errorf("write exception: write errors: [%s]", strings.Join(unorderedErrMsgs, ","))
+	// TODO: handle all errors and return them in slice
```

Comment:
I guess, this needs to be done here.

Reply:
I wanted to do this in separate PR because it will probably require to change some code in error.go